app/admin/dish.rb

```diff
@@ -0,0 +1,15 @@
+ActiveAdmin.register Dish do
+# See permitted parameters documentation:
```

Comment:
Cool. Is this where we will set permissions for different kinds of users? Like, admin can remove and add all dishes, but system user can only change the dishes that belongs to them?

Reply:
No that will be later, we need this https://www.pivotaltracker.com/story/show/151941930
to setup different type of permissions.